hnn_core/tests/test_gui.py

```diff
@@ -66,6 +66,9 @@ def test_gui_upload_params():
     gui._simulate_upload_connectivity(file2_url)
     # now connectivity is refreshed.
     assert gui.connectivity_widgets[0][0].children[1].value == 0.01
+
+    # We use -2 instead of -1 here as now the last one is the tonic inputs.
+    # changed back to -1 as tonic invalid inputs are removed.
```

Comment:
? I have no idea what this means :-)

Reply:
never mind, these are for myself and we can remove them now